src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -179,7 +179,8 @@ def load_command_table(self, _):
 
     compute_disk_encryption_set_sdk = CliCommandType(
         operations_tmpl='azure.mgmt.compute.operations#DiskEncryptionSetsOperations.{}',
-        client_factory=cf_disk_encryption_set
+        client_factory=cf_disk_encryption_set,
+        operation_group='disk_encryption_sets'
```

Comment:
typo?  added disk_encryption_sets group?

Reply:
good find. forgot to remove it.